android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -17,6 +19,8 @@
 class AppLovinMAXAdViewManager
         extends SimpleViewManager<AppLovinMAXAdView>
 {
+    public final int COMMAND_LOAD_AD = 1;
```

Comment:
Doesn't need to be `public` right? We should close the scope as much as possible for any fields. Also, static.
```suggestion
    private static final int COMMAND_LOAD_AD = 1;
```